bucket/vkquake.json

```diff
@@ -0,0 +1,80 @@
+{
+    "homepage": "https://github.com/Novum/vkQuake",
+    "description": "Quake 1 port using Vulkan instead of OpenGL for rendering, based on QuakeSpasm",
+    "version": "1.04.1",
+    "license": "GPL-2.0-or-later",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/Novum/vkQuake/releases/download/1.04.1/vkquake-1.04.1_win64.zip",
+            "hash": "9ad91717f12660fe9577bd8aa9695d3064069905a9bb82b8aeb7a45bc23cbf62",
+            "extract_dir": "vkquake-1.04.1_win64"
+        },
+        "32bit": {
+            "url": "https://github.com/Novum/vkQuake/releases/download/1.04.1/vkquake-1.04.1_win32.zip",
+            "hash": "c911ef84e6d44f42a2688d6c6ad12d0bb98cc21deda48f7382358dce83022e10",
+            "extract_dir": "vkquake-1.04.1_win32"
+        }
+    },
+    "shortcuts": [
+        [
+            "vkQuake.exe",
+            "vkQuake",
+            "-game id1"
+        ],
+        [
+            "vkQuake.exe",
+            "vkQuake (Scourge of Armagon)",
+            "-game hipnotic"
+        ],
+        [
+            "vkQuake.exe",
+            "vkQuake (Dissolution of Eternity)",
+            "-game rogue"
+        ],
+        [
+            "vkQuake.exe",
+            "vkQuake (Abyss of Pandemonium)",
+            "-game abyss"
+        ]
+    ],
+    "persist": [
+        "id1",
+        "hipnotic",
+        "rogue",
+        "abyss"
+    ],
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/Novum/vkQuake/releases/download/$version/vkquake-$version_win64.zip",
+                "extract_dir": "vkquake-$version_win64"
+            },
+            "32bit": {
+                "url": "https://github.com/Novum/vkQuake/releases/download/$version/vkquake-$version_win32.zip",
+                "extract_dir": "vkquake-$version_win32"
+            }
+        }
+    },
+    "notes": [
+        "vkQuake requires the Visual C++ Redistributable for Visual Studio 2019:",
```

Comment:
You can add `"depends": "extras/vcredist2019"` to the manifest so you don't have to tell the user manually in a note.

Reply:
Thanks for the suggestion. I didn't realize any Scoop buckets had the redists available. Addressed in latest commit.